## TALENTLAB ASSESSMENT

In [1]:
#import packages
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import scipy
from ast import literal_eval
import seaborn as sns
from scipy.stats import skew
from scipy.stats import mannwhitneyu

In [2]:
# Loading our dataset
Survey_df =pd.read_excel('../Dataset/TL20251023Mock Survey.xlsx')


In [3]:
Survey_df.head()

Submission Date         Region         Survey Type Respondent ID  \
0 2025-07-24 16:22:10.753    East Africa             Shelter      RESP1072   
1 2025-07-25 16:22:10.753    East Africa  Water & Sanitation      RESP1010   
2 2025-07-26 16:22:10.753    Middle East           Education      RESP1098   
3 2025-07-26 16:22:10.753  Latin America       Health Access      RESP1100   
4 2025-07-27 16:22:10.753     South Asia     He  alth Access      RESP1042   

              Gender Age Group  Q1: I have access to basic health services  \
0  Prefer not to say     18-24                                           5   
1  Prefer not to say     25-34                                           4   
2             Female  Under 18                                           3   
3         Non-binary     25-34                                           5   
4  Prefer not to say     25-34                                           5   

   Q2: My household has enough food to eat  \
0                                        5   
1                                        4   
2                                        2   
3                                        2   
4                                        5   

   Q3: My children can attend school safely  Q4: I feel safe in my shelter  \
0                                         5                              2   
1                                         2                              5   
2                                         3                              5   
3                                         1                              4   
4                                         4                              2   

   Q5: I have access to clean water  
0                                 1  
1                                 3  
2                                 5  
3                                 5  
4                                 5

In [4]:
# columns for the dataset
Survey_df.columns

Index(['Submission Date', 'Region', 'Survey Type', 'Respondent ID', 'Gender',
       'Age Group', 'Q1: I have access to basic health services',
       'Q2: My household has enough food to eat',
       'Q3: My children can attend school safely',
       'Q4: I feel safe in my shelter', 'Q5: I have access to clean water'],
      dtype='object')

In [5]:
Survey_df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 130 entries, 0 to 129
Data columns (total 11 columns):
 #   Column                                      Non-Null Count  Dtype         
---  ------                                      --------------  -----         
 0   Submission Date                             130 non-null    datetime64[ns]
 1   Region                                      130 non-null    object        
 2   Survey Type                                 130 non-null    object        
 3   Respondent ID                               130 non-null    object        
 4   Gender                                      130 non-null    object        
 5   Age Group                                   130 non-null    object        
 6   Q1: I have access to basic health services  130 non-null    int64         
 7   Q2: My household has enough food to eat     130 non-null    int64         
 8   Q3: My children can attend school safely    130 non-null    int64         
 9   Q4: I feel

In [6]:
# Retrieving unique values for dataframe
for column in Survey_df:
        print(f"Column Name : {column}")
        print(Survey_df [column].unique())
        print("-----------------------------")

Column Name : Submission Date
<DatetimeArray>
['2025-07-24 16:22:10.753000', '2025-07-25 16:22:10.753000',
 '2025-07-26 16:22:10.753000', '2025-07-27 16:22:10.753000',
 '2025-07-28 16:22:10.753000', '2025-07-29 16:22:10.753000',
 '2025-07-30 16:22:10.753000', '2025-07-31 16:22:10.753000',
 '2025-08-01 16:22:10.753000', '2025-08-02 16:22:10.753000',
 '2025-08-03 16:22:10.753000', '2025-08-05 16:22:10.753000',
 '2025-08-07 16:22:10.753000', '2025-08-08 16:22:10.753000',
 '2025-08-09 16:22:10.753000', '2025-08-10 16:22:10.753000',
 '2025-08-12 16:22:10.753000', '2025-08-13 16:22:10.753000',
 '2025-08-15 16:22:10.753000', '2025-08-16 16:22:10.753000',
 '2025-08-17 16:22:10.753000', '2025-08-18 16:22:10.753000',
 '2025-08-19 16:22:10.753000', '2025-08-20 16:22:10.753000',
 '2025-08-21 16:22:10.753000', '2025-08-22 16:22:10.753000',
 '2025-08-25 16:22:10.753000', '2025-08-26 16:22:10.753000',
 '2025-08-27 16:22:10.753000', '2025-08-29 16:22:10.753000',
 '2025-08-31 16:22:10.753000', '2025-09

## DATETIME

In [7]:
# Change to date
Survey_df['Submission Date'] = Survey_df['Submission Date'].dt.date


#### Survey Type

In [9]:
#Survey_df['Survey Type'] = Survey_df['Survey Type'].str.strip().str.replace(r'\s+', ' ', regex=True)

Survey_df['Survey Type'] = Survey_df['Survey Type'].replace(
    {'He  alth Access': 'Health Access',
    'Sh  elter': 'Shelter',
    'Ed  ucation':'Education',
    'Wa  ter & Sanitation': 'Water & Sanitation',
    'Fo  od Security':'Food Security',
    'Ed    ucation': 'Education'}
)

In [10]:
# unique values in Survey Type values
Survey_df['Survey Type'].unique()

array(['Shelter', 'Water & Sanitation', 'Education', 'Health Access',
       'Food Security'], dtype=object)

#### GENDER

In [11]:
# Replacing values in Gender values
Survey_df['Gender'] = Survey_df['Gender'].replace({
    'Non-binary': 'Other',
    'Prefer not to say': 'Other'
})

In [12]:
# unique values in gender values
Survey_df['Gender'].unique()

array(['Other', 'Female', 'Male'], dtype=object)

In [13]:
# Retrieving unique values for dataframe
for column in Survey_df:
        print(f"Column Name : {column}")
        print(Survey_df [column].unique())
        print("-----------------------------")

Column Name : Submission Date
[datetime.date(2025, 7, 24) datetime.date(2025, 7, 25)
 datetime.date(2025, 7, 26) datetime.date(2025, 7, 27)
 datetime.date(2025, 7, 28) datetime.date(2025, 7, 29)
 datetime.date(2025, 7, 30) datetime.date(2025, 7, 31)
 datetime.date(2025, 8, 1) datetime.date(2025, 8, 2)
 datetime.date(2025, 8, 3) datetime.date(2025, 8, 5)
 datetime.date(2025, 8, 7) datetime.date(2025, 8, 8)
 datetime.date(2025, 8, 9) datetime.date(2025, 8, 10)
 datetime.date(2025, 8, 12) datetime.date(2025, 8, 13)
 datetime.date(2025, 8, 15) datetime.date(2025, 8, 16)
 datetime.date(2025, 8, 17) datetime.date(2025, 8, 18)
 datetime.date(2025, 8, 19) datetime.date(2025, 8, 20)
 datetime.date(2025, 8, 21) datetime.date(2025, 8, 22)
 datetime.date(2025, 8, 25) datetime.date(2025, 8, 26)
 datetime.date(2025, 8, 27) datetime.date(2025, 8, 29)
 datetime.date(2025, 8, 31) datetime.date(2025, 9, 1)
 datetime.date(2025, 9, 3) datetime.date(2025, 9, 4)
 datetime.date(2025, 9, 6) datetime.date(2025

In [14]:
Survey_df.head()

Submission Date         Region         Survey Type Respondent ID  Gender  \
0      2025-07-24    East Africa             Shelter      RESP1072   Other   
1      2025-07-25    East Africa  Water & Sanitation      RESP1010   Other   
2      2025-07-26    Middle East           Education      RESP1098  Female   
3      2025-07-26  Latin America       Health Access      RESP1100   Other   
4      2025-07-27     South Asia       Health Access      RESP1042   Other   

  Age Group  Q1: I have access to basic health services  \
0     18-24                                           5   
1     25-34                                           4   
2  Under 18                                           3   
3     25-34                                           5   
4     25-34                                           5   

   Q2: My household has enough food to eat  \
0                                        5   
1                                        4   
2                                        2   
3                                        2   
4                                        5   

   Q3: My children can attend school safely  Q4: I feel safe in my shelter  \
0                                         5                              2   
1                                         2                              5   
2                                         3                              5   
3                                         1                              4   
4                                         4                              2   

   Q5: I have access to clean water  
0                                 1  
1                                 3  
2                                 5  
3                                 5  
4                                 5

In [15]:
# Checking missing values
Survey_df.isna().sum()

Submission Date                               0
Region                                        0
Survey Type                                   0
Respondent ID                                 0
Gender                                        0
Age Group                                     0
Q1: I have access to basic health services    0
Q2: My household has enough food to eat       0
Q3: My children can attend school safely      0
Q4: I feel safe in my shelter                 0
Q5: I have access to clean water              0
dtype: int64

In [16]:
# Checking for duplicates
Survey_df.duplicated().sum()

np.int64(10)

In [17]:
# Checking for duplicates
Survey_df[Survey_df.duplicated()]

Submission Date         Region         Survey Type Respondent ID  Gender  \
120      2025-08-21     South Asia           Education      RESP1046   Other   
121      2025-08-22    East Africa       Health Access      RESP1085  Female   
122      2025-07-27     South Asia       Health Access      RESP1042   Other   
123      2025-08-31    Middle East       Health Access      RESP1097   Other   
124      2025-08-09  Latin America             Shelter      RESP1015   Other   
125      2025-09-08    Middle East             Shelter      RESP1047  Female   
126      2025-09-15  Latin America             Shelter      RESP1054   Other   
127      2025-07-30     South Asia             Shelter      RESP1077  Female   
128      2025-08-19    Middle East           Education      RESP1019  Female   
129      2025-10-12  Latin America  Water & Sanitation      RESP1118   Other   

    Age Group  Q1: I have access to basic health services  \
120     25-34                                           5   
121     35-44                                           1   
122     25-34                                           5   
123  Under 18                                           4   
124     25-34                                           2   
125     35-44                                           5   
126     35-44                                           1   
127       55+                                           1   
128     25-34                                           2   
129  Under 18                                           3   

     Q2: My household has enough food to eat  \
120                                        1   
121                                        3   
122                                        5   
123                                        3   
124                                        4   
125                                        1   
126                                        3   
127                                        2   
128                                        5   
129                                        1   

     Q3: My children can attend school safely  Q4: I feel safe in my shelter  \
120                                         5                              2   
121                                         3                              5   
122                                         4                              2   
123                                         3                              5   
124                                         3                              5   
125                                         4                              4   
126                                         3                              1   
127                                         5                              1   
128                                         5                              3   
129                                         4                              4   

     Q5: I have access to clean water  
120                                 1  
121                                 5  
122                                 5  
123                                 3  
124                                 3  
125                                 5  
126                                 3  
127                                 4  
128                                 2  
129                                 4

In [18]:
# Generate excel sheet from dataframe
Survey_df.to_excel("/Users/mac/Documents/TALENT-LAB-ASSESSMENT/Dataset/output.xlsx", index=False)

### OBSERVATION

- Identified and removed 10 duplicate entries during the data quality review to ensure dataset integrity.

- Confirmed that the dataset contained no missing values, so no imputation was required.

- Trimmed extra spaces in the Survey Type column to maintain consistency in categorical values.

- Recategorized gender responses, grouping Non-binary and Prefer not to say under a unified “Other” category for clearer analysis and reporting.

- Converted the survey response scale from numeric values (1–5) to descriptive Likert terms: Strongly Agree, Agree, Neutral, Disagree, and Strongly Disagree to improve interpretability and clarity.

In [19]:
# Loading our dataset
Surveyy_df = pd.read_excel('../Dataset/output.xlsx')

In [20]:
# Retrieve first 5 rows 
Surveyy_df.head()

Submission Date         Region         Survey Type Respondent ID  Gender  \
0      2025-07-24    East Africa             Shelter      RESP1072   Other   
1      2025-07-25    East Africa  Water & Sanitation      RESP1010   Other   
2      2025-07-26    Middle East           Education      RESP1098  Female   
3      2025-07-26  Latin America       Health Access      RESP1100   Other   
4      2025-07-27     South Asia       Health Access      RESP1042   Other   

  Age Group  Q1: I have access to basic health services  \
0     18-24                                           5   
1     25-34                                           4   
2  Under 18                                           3   
3     25-34                                           5   
4     25-34                                           5   

   Q2: My household has enough food to eat  \
0                                        5   
1                                        4   
2                                        2   
3                                        2   
4                                        5   

   Q3: My children can attend school safely  Q4: I feel safe in my shelter  \
0                                         5                              2   
1                                         2                              5   
2                                         3                              5   
3                                         1                              4   
4                                         4                              2   

   Q5: I have access to clean water  
0                                 1  
1                                 3  
2                                 5  
3                                 5  
4                                 5

In [33]:
# Retrieving unique values for dataframe
for column in Surveyy_df:
        print(f"Column Name : {column}")
        print(Surveyy_df [column].unique())
        print("-----------------------------")

Column Name : Submission Date
<DatetimeArray>
['2025-07-24 00:00:00', '2025-07-25 00:00:00', '2025-07-26 00:00:00',
 '2025-07-27 00:00:00', '2025-07-28 00:00:00', '2025-07-29 00:00:00',
 '2025-07-30 00:00:00', '2025-07-31 00:00:00', '2025-08-01 00:00:00',
 '2025-08-02 00:00:00', '2025-08-03 00:00:00', '2025-08-05 00:00:00',
 '2025-08-07 00:00:00', '2025-08-08 00:00:00', '2025-08-09 00:00:00',
 '2025-08-10 00:00:00', '2025-08-12 00:00:00', '2025-08-13 00:00:00',
 '2025-08-15 00:00:00', '2025-08-16 00:00:00', '2025-08-17 00:00:00',
 '2025-08-18 00:00:00', '2025-08-19 00:00:00', '2025-08-20 00:00:00',
 '2025-08-21 00:00:00', '2025-08-22 00:00:00', '2025-08-25 00:00:00',
 '2025-08-26 00:00:00', '2025-08-27 00:00:00', '2025-08-29 00:00:00',
 '2025-08-31 00:00:00', '2025-09-01 00:00:00', '2025-09-03 00:00:00',
 '2025-09-04 00:00:00', '2025-09-06 00:00:00', '2025-09-07 00:00:00',
 '2025-09-08 00:00:00', '2025-09-09 00:00:00', '2025-09-10 00:00:00',
 '2025-09-11 00:00:00', '2025-09-12 00:00:00

## QUESTIONS

1. Calculate the average Q1 score (“Access to basic health services”) per Region.

In [21]:
# Calculate the average Q1 score per region and sort from highest to lowest
Avg1_by_region = Surveyy_df.groupby("Region")["Q1: I have access to basic health services"].mean().reset_index().sort_values(by="Q1: I have access to basic health services", ascending=False)

print(Avg1_by_region)


          Region  Q1: I have access to basic health services
3     South Asia                                    3.419355
4    West Africa                                    3.000000
2    Middle East                                    2.826087
0    East Africa                                    2.739130
1  Latin America                                    2.615385


The analysis indicates notable regional differences in perceived access to basic health
services (Q1). South Asia reported the highest average score (3.4), suggesting
relatively better access. This is followed by West Africa (3.0) and the Middle East
(2.8). East Africa (2.7) and Latin America (2.6) showed the lowest averages,
highlighting potential gaps in health service availability in these regions.

2. Find which Age Group reports the lowest access to health services.

In [22]:
# Filter only "Disagree" and "Strongly Disagree" responses
low_access = Surveyy_df[Surveyy_df["Q1: I have access to basic health services"].isin([1, 2])]

In [23]:
# Count by Age Group
low_access_count = (
    low_access.groupby("Age Group")["Q1: I have access to basic health services"]
    .count()
    .reset_index(name="Count_Low_Access")
    .sort_values(by="Count_Low_Access", ascending=False)
)

print(low_access_count)

  Age Group  Count_Low_Access
2     35-44                15
1     25-34                11
4       55+                10
0     18-24                 7
3     45-54                 7
5  Under 18                 3


From our analysis, individuals who selected options 1 or 2 (representing Strongly Disagree and Disagree) indicate lower access to basic health services. The age group 35– 44 reported the highest number of respondents with limited access (15 people), followed by 25–34 (11 people), 55 and above (10 people), 18–24 (7 people), and 45–54 (7 people).

3. Determine the percentage of respondents who rated “5” for Q2 (Food security) across all regions.

In [24]:
# Count how many respondents in each region selected 5 in Q2
Q2_five_counts = Surveyy_df[Surveyy_df["Q2: My household has enough food to eat"] == 5] \
                    .groupby("Region")["Q2: My household has enough food to eat"].count()

In [25]:
# Count total respondents per region
total_counts = Surveyy_df.groupby("Region")["Q2: My household has enough food to eat"].count()

# Calculate percentage
Q2_five_percentage = (Q2_five_counts / total_counts) * 100

Q2_five_percentage = Q2_five_percentage.round(1)  # ← round to 1 decimal place

# Convert to a dataframe for readability
Q2_five_percentage = Q2_five_percentage.reset_index(name="Percentage_Rated_5")

Q2_five_percentage = Q2_five_percentage.sort_values(by="Percentage_Rated_5", ascending=False)

print(Q2_five_percentage)

          Region  Percentage_Rated_5
1  Latin America                19.2
2    Middle East                17.4
3     South Asia                16.1
0    East Africa                13.0
4    West Africa                 5.9


From our analysis, the Latin America recorded the highest percentage of respondents who selected “5” (indicating strong food security) at 19.2%. This is followed by Middle East at 17.4%, South Asia at 16.1%, East Africa at 13%, and West Africa with the lowest percentage at 5.9%.

4. Identify which Survey Type has the highest overall satisfaction (average across Q1–Q4).

In [28]:
# Calculate average satisfaction score across Q1–Q4 for each respondent
Surveyy_df["Avg_Satisfaction"] = Surveyy_df[[
    "Q1: I have access to basic health services",
    "Q2: My household has enough food to eat",
    "Q3: My children can attend school safely",
    "Q4: I feel safe in my shelter"
]].mean(axis=1)

# Group by Survey Type and calculate the average
survey_type_satisfaction = Surveyy_df.groupby("Survey Type")["Avg_Satisfaction"].mean().reset_index()

# Sort in descending order to find the highest satisfaction
survey_type_satisfaction = survey_type_satisfaction.sort_values(by="Avg_Satisfaction", ascending=False)

survey_type_satisfaction

Survey Type  Avg_Satisfaction
2       Health Access          3.086538
3             Shelter          3.032609
1       Food Security          2.820000
0           Education          2.773810
4  Water & Sanitation          2.760000

From the analysis, the health Access survey type recorded the highest overall satisfaction score at 3.08, followed closely by Shelter with 3.03. Food Security and Education followed with scores of 2.82 and 2.77 respectively, while Water & Sanitation had the lowest overall satisfaction score at 2.76.


5. Compare male vs. female respondents’ average scores in Q3 (Children attending school safely).


In [30]:
# Filter to only Male and Female
mf_df = Surveyy_df[Surveyy_df["Gender"].isin(["Male", "Female"])]

# Now calculate the average Q3 score
genderq3_comparison = mf_df.groupby("Gender")["Q3: My children can attend school safely"].mean().reset_index()

print(genderq3_comparison)


   Gender  Q3: My children can attend school safely
0  Female                                  2.909091
1    Male                                  3.045455


From the analysis, male respondents reported a slightly higher average satisfaction score (3.0) regarding the statement “My children can attend school safely” compared to female respondents, who reported an average score of 2.9. This suggests that men generally perceive school safety conditions more
positively than women.


6. Compute the correlation between access to health services (Q1) and food security (Q2).

In [32]:
# Compute the correlation between access to health services (Q1) and food security (Q2)
correlation = Surveyy_df["Q1: I have access to basic health services"].corr(
    Surveyy_df["Q2: My household has enough food to eat"]
)

# Display the correlation rounded to 3 decimal places
print("Correlation between Q1 and Q2 is:", round(correlation, 3))

Correlation between Q1 and Q2 is: 0.068


The correlation between access to health services (Q1) and food security (Q2) is 0.068, indicating a very weak positive relationship. This suggests that having better access to health services does not necessarily correspond to having better access to sufficient nutritious food among the respondents.

7. which region appears to have the highest overall quality of life?

In [46]:
# Calculate average satisfaction score across Q1–Q4 for each respondent
Surveyy_df["Avg_Satisfaction"] = Surveyy_df[[
    "Q1: I have access to basic health services",
    "Q2: My household has enough food to eat",
    "Q3: My children can attend school safely",
    "Q4: I feel safe in my shelter"
]].mean(axis=1)

# Group by Survey Type and calculate the average
Region_satisfaction = Surveyy_df.groupby("Region")["Avg_Satisfaction"].mean().reset_index()

# Sort in descending order to find the highest satisfaction
Region_satisfaction  = Region_satisfaction .sort_values(by="Avg_Satisfaction", ascending=False)

Region_satisfaction 

Region  Avg_Satisfaction
3     South Asia          3.153226
1  Latin America          2.961538
2    Middle East          2.804348
4    West Africa          2.720588
0    East Africa          2.706522

## SCENARIO / WHAT IF ANALYSIS

1. If Q1 scores increase by 1 point for respondents under 25, how does the overall average
change?

In [35]:
# Create a copy so we don’t overwrite original data
scenario_df = Surveyy_df.copy()

# Increase Q1 by +1 for respondents under age 25 (Under 18 and 18–24)
scenario_df.loc[scenario_df["Age Group"].isin(["Under 18", "18-24"]),
                "Q1: I have access to basic health services"] += 1


In [36]:
# Recalculate averages
original_avg = Surveyy_df["Q1: I have access to basic health services"].mean()
new_avg = scenario_df["Q1: I have access to basic health services"].mean()

print("Original Average Q1:", round(original_avg, 2))
print("New Average Q1 After Scenario:", round(new_avg, 2))
print("Change:", round(new_avg - original_avg, 2))


Original Average Q1: 2.94
New Average Q1 After Scenario: 3.22
Change: 0.28


After increasing Q1 scores for respondents aged under 25 (Under 18 and 18–24) by one point, the overall average Q1 score increased from 2.94 to 3.22, representing a change of 0.28.

Increasing Q1 scores by one point for respondents under 25 raised the overall average health access score from 2.94 to 3.22, a noticeable increase of 0.28. This indicates that improving access to basic health services for younger age groups has a meaningful positive impact on the overall population’s perceived access to healthcare.

2. Build a two-variable data table showing how Q2 and Q3 averages change with different regional weights.

In [38]:
# Calculate the average Q2 (food security) and Q3 (school safety) scores for each region
regional_avgs = Surveyy_df.groupby("Region")[
    ['Q2: My household has enough food to eat',
    'Q3: My children can attend school safely']
].mean().reset_index()

regional_avgs

Region  Q2: My household has enough food to eat  \
0    East Africa                                 2.652174   
1  Latin America                                 2.846154   
2    Middle East                                 2.869565   
3     South Asia                                 2.870968   
4    West Africa                                 2.470588   

   Q3: My children can attend school safely  
0                                  2.695652  
1                                  3.269231  
2                                  2.608696  
3                                  3.419355  
4                                  2.823529

In [42]:
weights = {
    "Equal": [1,1,1,1,1],                  # all equal
    "High Africa": [3,1,1,1,3],            # East + West Africa prioritized
    "High South Asia": [1,1,3,1,1],
    "High Middle East": [1,1,1,3,1]
}


In [43]:
# Extract region names and their corresponding average Q2 and Q3 values into lists
regions = regional_avgs["Region"].tolist()
Avg_Q2 = regional_avgs["Q2: My household has enough food to eat"].tolist()
Avg_Q3 = regional_avgs["Q3: My children can attend school safely"].tolist()

results = []

# Loop through each weighting scenario to compute weighted averages

for scenario, w in weights.items():
    total_weight = sum(w)

    # Compute weighted averages for Q2 and Q3 based on selected weight pattern

    weighted_Q2 = np.average(Avg_Q2, weights=w)
    weighted_Q3 = np.average(Avg_Q3, weights=w)
    results.append([scenario, round(weighted_Q2, 2), round(weighted_Q3, 2)])


# Convert results into a DataFrame for easy viewing
data_table = pd.DataFrame(results, columns=["Scenario", "Weighted Avg Q2", "Weighted Avg Q3"])

# Display final weighted scenario comparison table
print(data_table)


           Scenario  Weighted Avg Q2  Weighted Avg Q3
0             Equal             2.74             2.96
1       High Africa             2.66             2.87
2   High South Asia             2.78             2.86
3  High Middle East             2.78             3.09


When all regions are weighted equally, the overall food security (Q2) and school safety (Q3) averages are 2.74 and 2.96, respectively. Prioritizing East and West Africa (High Africa scenario) lowers the overall averages to 2.66 for Q2 and 2.87 for Q3. This occurs because these African regions have comparatively lower scores, so giving them more weight pulls down the global averages. Conversely, prioritizing regions with higher scores, such as the Middle East (High Middle East scenario), increases the overall Q3 average to 3.09. These results illustrate that regional disparities significantly influence overall outcomes, and targeted weighting can reveal which regions drive the global metrics up or down.